src/newfiledialog.cpp

```diff
@@ -90,11 +90,22 @@ void NewFileDialog::on_loadFileButton_clicked()
 {
     // Check that there is a file selected
     QString fname = ui->newFileEdit->text();
-    if (fname.isEmpty()) {
+    QFileInfo checkfile(fname);
+    if (!checkfile.exists() && !checkfile.isFile()) {
```

Comment:
Indeed, but both cases separately are also stoppers, not just both of them at the same time. Meaning, if it is not a file or it doesn't exist, in an case, user should select a new one, right?